package.json

```diff
@@ -25,7 +25,7 @@
     "node",
     "setup"
   ],
-  "author": "",
+  "author": "Keith Zantow <kzantow@gmail.com>",
```

Comment:
Since this is an Anchore project, we should probably only use Anchore-managed email addresses